docs/nodes/blockspace-race.mdx

```diff
@@ -191,6 +191,7 @@ provided by the community here.
 * [http://celestia-itn.staking4all.org:26657](http://celestia-itn.staking4all.org:26657)
 * [https://celestia-testnet-rpc.nodeguardians.io/](https://celestia-testnet-rpc.nodeguardians.io/)
 * [https://rpc-blockspacerace.staker.space/](https://rpc-blockspacerace.staker.space/)
+* [http://rpc-blockspacerace.freshstaking.com:52657](http://rpc-blockspacerace.freshstaking.com:52657/)
```

Comment:
```suggestion
* [http://rpc-blockspacerace.freshstaking.com:52657/](http://rpc-blockspacerace.freshstaking.com:52657/)
```

Reply:
oopps, apologies. I am a bit of nit also but I missed this one!